server/sample.env

```diff
@@ -1,3 +1,8 @@
 JWT_SECRET=
 MONGO_URI=
-TEAM_NAMES=Bonnie,Sina,Ankit,Peter,Uche,Antwi
\ No newline at end of file
+TEAM_NAMES=Bonnie,Sina,Ankit,Peter,Uche,Antwi
+PORT=
+CLOUDINARY_CLOUD=
+CLOUDINARY_API_KEY=
+CLOUDINARY_API_SECRET=
+SENDGRID_API_KEY=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
```

Comment:
**Major security breach here, regenerate your API key and remove this from the `sample.env`**

Reply:
You will have to revert the change in that file. And not just remove the key as that will still expose it in the diffs.